Library/Homebrew/dev-cmd/bottle.rb

```diff
@@ -190,7 +190,10 @@ def bottle_formula(f)
       rebuild = rebuilds.empty? ? 0 : rebuilds.max.to_i + 1
     end
 
-    compression_type = f.bottle_specification.compression_type
+    compression_type = :xz
```

Comment:
You're going to have to carve out an exemption here for the `xz` formula itself, at the very least on Linux, presumably. Otherwise you're going to end up with the `xz` bottle compressed with `xz`, which is problematic.

Reply:
😆 You beat me to it. I'll shut up and let ya do your thing for a bit!